frontend/app/controllers/spree/orders_controller.rb

```diff
@@ -92,7 +97,7 @@ def accurate_title
 
     def check_authorization
       order = Spree::Order.find_by(number: params[:id]) if params[:id].present?
```

Comment:
`order = Spree::Order.find_by(number: params[:id]) || current_order` should do the job. drop this `if` statement and next line

Reply:
Okay I changed it.